llvm/lib/Target/RISCV/MCTargetDesc/RISCVMCCodeEmitter.cpp

```diff
@@ -400,6 +404,23 @@ RISCVMCCodeEmitter::getImmOpValueAsr1(const MCInst &MI, unsigned OpNo,
   return getImmOpValue(MI, OpNo, Fixups, STI);
 }
 
+uint64_t
+RISCVMCCodeEmitter::getImmOpValueZibimm(const MCInst &MI, unsigned OpNo,
+                                        SmallVectorImpl<MCFixup> &Fixups,
+                                        const MCSubtargetInfo &STI) const {
+  const MCOperand &MO = MI.getOperand(OpNo);
+
+  if (MO.isImm()) {
+    int64_t Res = MO.getImm();
+    if (Res == -1)
+      return 0;
+
+    return Res;
+  }
+
+  return getImmOpValue(MI, OpNo, Fixups, STI);
```

Comment:
Will we ever get here? If it's not an immediate we don't have a relocation to encode the cimm correctly. So I this isImm must always be true?